### ISSUE TITLE
LCIO output doesn't work

### ISSUE THREAD

**Issue by simonge** [eic/ip6#51]
### Environment: (where does this bug occur, have you tried other environments)
All tested environments (eic-shell and local install) and branches (main and 43)

### Steps to reproduce: (give a step by step account of how to trigger the bug)
Run ddsim with ip6.xml with no output/lcio output specified.
e.g.
ddsim --compactFile ip6.xml --enableGun --numberOfEvents 10
or
ddsim --compactFile ip6.xml --enableGun --numberOfEvents 10 --outputFile temp.slcio

### Expected Result: (what do you expect when you execute the steps above)
LCIO output file with simulation results

### Actual Result: (what do you get when you execute the steps above)
segfault
On eic-shell begins:
### CAUGHT SIGNAL: 11 ### address: 0x8,  signal =  SIGSEGV, value =   11, description = segmentation violation. Address not mapped to object.

I'm not using this myself however it would be sensible to assume it should work as it does with the epic.xml

**Comment by demisra** [comment]
Are you trying to use the ip6.xml located in the ip6 directory? You need to use epic_ip6.xml located in your $DETECTOR_PATH to correctly load the geometry. I can reproduce this error using ip6/ip6.xml, so I presume that's what you're doing here.

**Comment by simonge** [comment]
That is correct but ideally it should still be possible to run the ip6 geometry identically and independently of epic.

**Comment by veprbl** [comment]
Can you post the backtrace?

**Comment by simonge** [comment]
This will hopefully become irrelevant after the codes are merged.
